docs/getting-started/fundamentals-block-development/file-structure-of-a-block.md

```diff
@@ -0,0 +1,79 @@
+# File structure of a block
+
+At its simplest, a block in the WordPress block editor is a JSON object with a specific set of properties that is registered in the system so it can be used.
```

Comment:
This is incorrect. At its simplest it is a registerBlockType function call with some parameters passed in JS

Block.json is just an abstraction that makes this easier and allows us to more easily share the configuration between JS and PHP.

Reply:
Thanks a lot for this feedback @fabiankaegy 🙌
I have adjusted the sentences in this page to take this into account.